### PR TITLE
UML diagram For Issues #53 #29 #7

### LINKED ISSUE #7: Student - Share Shortlists
In this Epic, students share their lists they have made with their parents to get feedback and have discussions with their parents.

### LINKED ISSUE #29: Student & Parent - Categorize recommended High Schools
In this epic, Students and Parents will be able to categorize High Schools that Shortlist has recommended for them to review.

### LINKED ISSUE #53: Parents - Recommend Chosen Schools to Students
Parents allow the app to use the schools they have chosen in their lists to influence Students' choices.

### PR DESCRIPTION
**Pull request by simaoalicechen**
Added UML for Issues #53 #29 #7: 

- Students and Parents Categorize High Schools/Make Lists
- Students Share ShortLists
- Parents Recommends High Schools

Please review.

### CODE REVIEW

**Review by yalexublue — CHANGES_REQUESTED** [review]
Consider further separate epics to have less elements in one to have a clearer view. Also please use actor components for user n also activation, example has been sent in slack channel last night.

**Comment by jainsvaibhav** [comment]
> Consider further separate epics to have less elements in one to have a clearer view. Also please use actor components for user n also activation, example has been sent in slack channel last night.

Yeah I think we can keep the parent persona in one UML and the child in another

**Comment by jainsvaibhav** [comment]
We can also separate this UML based on Categorizing and Sharing into 2 different UML. Once things are categorized, we can put an arrow saying "share with parent" epic.

**Comment by jainsvaibhav** [comment]
can u please restructure the files into docs/design/file.ext

**Comment by simaoalicechen** [comment]
Updated

**Comment by yalexublue** [comment]
> Updated
What Alex meant was use docs/design/diagram.png, diagram.txt format, you can check develop branch's file structure, other than that I think new diagrams structure looks great, I will add detailed comment to them, I don't think there are any major issues

**Review by yalexublue — CHANGES_REQUESTED** [review]
#7 For this diagram, the first UI activate time is a bit too short, imo it should stay alove till after make personal notes on each school, then for the loop section, when parents click into a school and start a discussion, it's not a feedback thus you don't want dashed line instead should use solid lines. Also I don't think mentioning UI would be wise here as this process would definitely include API n database. So I would suggest use SHortlist as participant only. Finally, you don't activate on Users, Here in the txt is what I meant, it's based on what you currently have you could take a look
[#7-(Students)Share-ShortLists-with-their-Parents.txt](https://github.com/gcivil-nyu-org/team-2-inperson/files/9708866/7-.Students.Share-ShortLists-with-their-Parents.txt)

File: docs/design/#29 Updated.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title #29 (Students&Parents) Categorize High Schools and Make Lists
```
Comment (Pooja-Codes) [inline docs/design/#29 Updated.txt]: We are not going to mention the epic tags or names on the high-level design page. A suggestion would be to change the title.

File: docs/design/#53 Updated.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+title #29 (Students&Parents) Categorize High Schools and Make Lists
```
Comment (Pooja-Codes) [inline docs/design/#53 Updated.txt]: Same the title change. Great work!

File: docs/design/#7 Updated.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+title #7 (Students) Share ShortLists with their Parents
```
Comment (Pooja-Codes) [inline docs/design/#7 Updated.txt]: Great Flow! Just title change.

**Review by Pooja-Codes — CHANGES_REQUESTED** [review]
Title changes require!

**Comment by Pooja-Codes** [comment]
Also, Can you remove the fork of the team2 repo?

**Review by jainsvaibhav — APPROVED** [review]
LGTM

**Pull request merged**